steps.py

```diff
@@ -367,48 +376,90 @@ class FLESteps(Stepper):
     def __init__(
         self,
         step_batchsize: int = 200,
-        gamma: float = 0.015,
+        gamma: float = 0.00375,
         alpha: float = 0.448,
+        bound_gamma: float = 0.00075,
+        bound_alpha: float = 0.373,
         dt: float = 0.210,
         fle_random_seed: int = None,
-        boundstepper: Stepper = None
+        bound_steps: str = "FLE",
     ):
 
-        self.H = alpha / 2
         self.gamma = gamma
         self.alpha = alpha
+        self.bound_gamma = bound_gamma
+        self.bound_alpha = bound_alpha
         self.dt = dt
         self.cur_step = 0
+        self.real_step = 0
         self.step_batchsize = step_batchsize
-
-        self.pre_x, self.pre_y = self._generate_correlated_noise(
+        self.bound_steps = bound_steps
+        (
+            self.pre_x,
+            self.pre_y,
+            self.bound_pre_x,
+            self.bound_pre_y,
+        ) = self._generate_correlated_noise(
```

Comment:
Yes, it is important to always regenerate both bound and unbound steps to ensure they are both based on the same underlying correlated noise. We could re-write to only generate correlated noise, then perform normalization on the desire step when a step is taken. This would be a good change if we start to include more than movement regimes in a single trajectory.